apis/Google.Cloud.Channel.V1/docs/history.md

```diff
@@ -1,5 +1,16 @@
 # Version history
 
+# Version 1.0.0, released 2021-03-03
+
+- [Commit 1620a3a](https://github.com/googleapis/google-cloud-dotnet/commit/1620a3a): feat: addition of billing_account field on Plan. docs: clarification that valid address lines are required for all customers.
```

Comment:
nit: there's a feat: and a docs: on this comment, without a line break or bullets, the ones in the next commit are in bullets.

Reply:
Well spotted. I manually fixed the other one - will do this one too.